demo/src/main/java/com/orange/ods/demo/ui/components/textfields/SubComponentTextFieldFilled.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ *
+ *  Copyright 2021 Orange
+ *
+ *  Use of this source code is governed by an MIT-style
+ *  license that can be found in the LICENSE file or at
+ *  https://opensource.org/licenses/MIT.
+ * /
+ */
+
+package com.orange.ods.demo.ui.components.textfields
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import com.orange.ods.compose.component.textfield.OdsTextField
+import com.orange.ods.demo.R
+import com.orange.ods.demo.ui.components.utilities.clickOnElement
+
+@ExperimentalMaterialApi
+@Composable
+fun TextFieldFilledContent(customizationState: TextFieldCustomizationState) {
+    var text by rememberSaveable { mutableStateOf("") }
+    val context = LocalContext.current
+    val trailingIconName = stringResource(id = R.string.component_element_trailing)
+
+    OdsTextField(
```

Comment:
Should we rename it to `OdsFilledTextField` ?

Reply:
I called it `OdsTextField` to match with the framework composables names. The Jetpack Compose composables  are `TextField` and `OutlinedTextField`.